taxi/models.py

```diff
@@ -10,10 +10,12 @@ class Manufacturer(models.Model):
 class Driver(AbstractUser):
     license_number = models.CharField(max_length=255, unique=True)
 
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
 
 class Car(models.Model):
     model = models.CharField(max_length=255)
-    manufacturer = models.ForeignKey(
-        Manufacturer, on_delete=models.CASCADE, related_name="cars"
-    )
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
```

Comment:
Add a `related_name` to this field. Besides that, add `__str__` methods to the models.